pkg/statistics/handle/storage/read.go

```diff
@@ -843,3 +843,35 @@ func StatsMetaByTableIDFromStorage(sctx sessionctx.Context, tableID int64, snaps
 	count = rows[0].GetInt64(2)
 	return
 }
+
+// convertBoundFromBlob reads the bound from blob. The `blob` is read from the `mysql.stats_buckets` table.
+// The `convertBoundFromBlob(convertBoundToBlob(a))` should be equal to `a`.
+// TODO: add a test to make sure that this assumption is correct.
```

Comment:
BTW, I think the stats need a more stable serialization/deserialization method. Depending on `ConvertTo` to/from `BLOB` is not good in long term 🤦 .